umbral/fragments.py

```diff
@@ -53,31 +53,20 @@ def to_bytes(self):
     def verify(self, pub_a, pub_b, params: "UmbralParameters"=None):
         params = params if params is not None else default_params()
 
+        u = params.u
+
         u1 = self.point_commitment
         z1 = self.bn_sig1
         z2 = self.bn_sig2
         x = self.point_eph_ni
+        key = self.bn_key
 
         g_y = (z2 * params.g) + (z1 * pub_a)
 
-        return z1 == hash_to_bn([g_y, self.bn_id, pub_a, pub_b, u1, x], params)
-
-    def is_consistent(self, vKeys, params: "UmbralParameters"=None):
-        params = params if params is not None else default_params()
-
-        if vKeys is None or len(vKeys) == 0:
-            raise ValueError('vKeys must not be empty')
-
-        h = params.h
-        lh_exp = self.bn_key * h
-
-        rh_exp = vKeys[0]
-        i_j = self.bn_id
-        for vKey in vKeys[1:]:
-            rh_exp = rh_exp + (i_j * vKey)
-            i_j = i_j * self.bn_id
+        check_kfrag_1 = z1 == hash_to_bn([g_y, self.bn_id, pub_a, pub_b, u1, x], params)
```

Comment:
I think I need a comment here.  Why do I expect the hash to be the same as `z1`?

Reply:
Done. Basically this is just the check performed to validate a Schnorr signature